controllers/util/mw_util.go

```diff
@@ -261,44 +263,51 @@ func (mwu *MWUtil) ClusterRolesDelete(drpolicy *rmn.DRPolicy) error {
 	return nil
 }
 
-func (mwu *MWUtil) createOrUpdateClusterRolesManifestWork(namespace string) error {
-	manifestWork, err := mwu.generateClusterRolesManifestWork(namespace)
+func (mwu *MWUtil) createOrUpdateDrClusterManifestWork(namespace string, ramenConfig *rmn.RamenConfig) error {
+	manifestWork, err := mwu.generateDrClusterManifestWork(namespace, ramenConfig)
 	if err != nil {
 		return err
 	}
 
 	return mwu.createOrUpdateManifestWork(manifestWork, namespace)
 }
 
-func (mwu *MWUtil) generateClusterRolesManifestWork(namespace string) (*ocmworkv1.ManifestWork, error) {
-	vrgClusterRole, err := mwu.generateVRGClusterRoleManifest()
-	if err != nil {
-		mwu.Log.Error(err, "failed to generate VolumeReplicationGroup ClusterRole manifest")
-
-		return nil, err
-	}
-
-	vrgClusterRoleBinding, err := mwu.generateVRGClusterRoleBindingManifest()
-	if err != nil {
-		mwu.Log.Error(err, "failed to generate VolumeReplicationGroup ClusterRoleBinding manifest")
-
-		return nil, err
-	}
+func (mwu *MWUtil) generateDrClusterManifestWork(
+	clusterName string,
+	ramenConfig *rmn.RamenConfig,
+) (*ocmworkv1.ManifestWork, error) {
+	objects := [...]interface{}{
+		vrgClusterRole,
+		vrgClusterRoleBinding,
+		namespace(ramenConfig.DrClusterOperator.NamespaceName),
+		olmClusterRole,
+		olmRoleBinding(ramenConfig.DrClusterOperator.NamespaceName),
+		operatorGroup(ramenConfig.DrClusterOperator.NamespaceName),
+		catalogSource(ramenConfig.DrClusterOperator.NamespaceName, ramenConfig.DrClusterOperator.CatalogSourceImageName),
```

Comment:
We do not want to install the catalog. We will require that are a pre-req for the user to install required catalogs on the managed clusters.

This would also clear out any roles created for the catalog etc. and its settings in the config map.

Reply:
Ok.

Reply:
Changes to remove catalog source deployment from ramen hub were made in https://github.com/hatfieldbrian/ramen/commit/b53c30c0f239bc1a734f42b74a865c21bfe5dbe9.  End-to-end test script now deploys and undeploys it.  `olm-edit` role was left for `OperatorGroup` deployment.